app/util.py

```diff
@@ -39,6 +40,14 @@ def recordAPIStatus(apiName):
             if (r.status_code == 429):
                 e_code = r.json().get("code")
                 req = False
+        elif (apiName == "tripadvisor-mapper"):
```

Comment:
nit: no parens needed for if. Save time: don't change it now, but for future reference.